src/style.css

```diff
@@ -0,0 +1,105 @@
+:root {
+  --padding-left-big-search: 62px;
+  --padding-left-small-search: 33px;
+  --big-search-height: 70px;
+  --small-search-height: 42px;
+  --size-big-icon: 19px;
+  --size-small-icon: 10px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  position: relative;
+  height: var(--big-search-height);
+  margin-bottom: 20px;
+}
+
+.forms-wrapper {
+  margin-top: 20px;
+}
+
+.label-search-icon {
+  position: absolute;
+  cursor: text;
+
+}
+
+.label-search-icon,
```

Comment:
Double check icon style as on DEMO it looks like this:
<img width="458" alt="image" src="https://user-images.githubusercontent.com/126275137/233351240-bdea739c-4b0b-46b0-8059-a835b2c7b6f7.png">